frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/__tests__/utils.spec.ts

```diff
@@ -1,12 +1,12 @@
 import * as _ from 'lodash';
 import { pipelineParameters, pipelineParametersWithoutDefaults } from './utils-data';
-import { PipelineParam } from '../../../../utils/pipeline-augment';
+import { TektonParam } from '../../../../types';
```

Comment:
Is there any specific reason we are using `Tekton` in type name here? I only ask as I don't see it anywhere else. 
Could we make use of something more relatable like `CommonParam` WDYT?

Reply:
Because it's generic. I am open to another name... but both Task and Pipeline use the same Parameter type... so I suppose I could called it TektonParam and then create stand alone types for Pipelines and Tasks for "their params" but it seemed kinda needless.

Reply:
cool!!